### DISCUSSION TITLE
Exclude the backspace key from backspacing the last space in the sentence if used as a hotkey for "clear single word" feature in Chat Commands plugin

### DISCUSSION THREAD

**Discussion by IEarnSolo** [runelite/runelite#14940] [Ideas]
Edit: The man has spoken and I don't know why I am this way
![image](https://user-images.githubusercontent.com/105403369/167983275-8d6f5c58-2000-439f-bc14-ee0587b95d39.png)

The backspace key should be excluded from removing the space at the end of the sentence if backspace is used as a hotkey for "clear single word" feature in Chat Commands plugin. So I have ctrl+backspace set to clear one word at a time, as it typically is the hotkey for every other program globally on your computer. However because I'm hitting backspace, it will clear the word and then also backspace the last space at the end of the sentence which is not what normally happens when using this feature in any other program in windows (probably mac too). This is only caused if the backspace is used as the hotkey, otherwise it works fine like with the default ctrl+w hotkey (which is an odd default hotkey in my opinion) 

If this were fixed that would be awesome because:

https://user-images.githubusercontent.com/105403369/167982366-44d290d4-e23a-4167-b8cd-735be5421dec.mp4

It always causes me to type into my previous word because there's no space, so I hit ctrl+backspace again, and even more of my sentence is gone than I intended.

**Comment by Nightfirecat** [comment]
This appears to be a bug, I've opened #14941 to track it.

**Reply by Nightfirecat** [reply]
This is fixed as of release 1.8.19.1
